### PR TITLE
ITI approved CP integration into TF volume 2

### LINKED ISSUE #295: Integrate ITI approved CPs into Volume 2
793-06
1271-02
1278-02 (done previously)
959-01
1150-02
1226-04
1272-04
1289-00
1167-01
1273-03
1254-03
1260-03
1219-04

### PR DESCRIPTION
**Pull request by JohnMoehrke**
close #295

### CODE REVIEW

File: ITI/TF/Volume2/ITI-21.html [hunk 1]
```diff
@@ -1155,7 +1155,11 @@ <h5 id="3.21.5.1.1">3.21.5.1.1 Patient Demographics Consumer audit message:</h5>
           <td>not specialized</td>
         </tr>
         <tr>
-          <td>ParticipantObjectID</td>
```
[the remark this answers is not in the document]
Comment (JohnMoehrke) [reply ITI/TF/Volume2/ITI-21.html:1158]: good catch. Fixed.

File: ITI/TF/Volume2/ITI-38.html [hunk 2]
```diff
@@ -317,24 +317,33 @@ <h5 id="3.38.4.1.3">3.38.4.1.3 Expected Actions</h5>
         into
         the response to the Cross Gateway Query.</li>
       <li class="bullet-list1">Ensure that the response contains a value identifying one of the
-        Responding Gateway’s communities for the homeCommunityId attribute in every appropriate
+        Responding Gateway's communities for the homeCommunityId attribute in every appropriate
         element. The elements that shall include the ebRIM home attribute are:</li>
-    </ul>
```
[the remark this answers is not in the document]
Comment (JohnMoehrke) [reply ITI/TF/Volume2/ITI-38.html:322]: @gregoriocanal do you agree?

**Comment by gregoriocanal** [comment]
Yes i agree

Il mar 4 lug 2023, 15:41 John Moehrke ***@***.***> ha scritto:

**Review by gregoriocanal — APPROVED** [review]

**Comment by gregoriocanal** [comment]
I have approved the review.
Am i supposed to do the merge ?

Il mer 5 lug 2023, 14:01 John Moehrke ***@***.***> ha scritto:

**Comment by JohnMoehrke** [comment]
approve is all we need at this time.

John Moehrke 🔥 Architect: Healthcare Informatics Standards -
Interoperability, Privacy, and Security
IHE Co-Chair IT Infrastructure Planning and Technical
HL7 Co-Chair Security WG, FHIR FMG, FHIR facilitator, and FHIR Foundation
founding member
Employee of By Light -- Contractor to VHA MyHealtheVet
***@***.***  |  M +1 920-564-2067  |  ***@***.***
 https://healthcaresecprivacy.blogspot.com

**Pull request merged**